OWNERS_ALIASES

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#owners_aliases
+
+aliases:
+  mustgather-approvers:
```

Comment:
+1 thanks, can you also add [1] to the list of the `must-gather` approvers? 

[1] https://github.com/Akrog/